portal/models/communication.py

```diff
@@ -127,7 +128,13 @@ def _lookup_questionnaire_due_date():
         trigger_date = qb.trigger_date(user)
         due = (qb.calculated_overdue(trigger_date) or
                qb.calculated_expiry(trigger_date))
-        return due.strftime('%-d %b %Y') if due else ''
+        due_date = ''
+        if user.timezone and due:
+            local = utc.localize(due).astimezone(timezone(user.timezone))
```

Comment:
Can we make this a function on user or perhaps better in `date_tools`?  I would expect there are several places where we'll need to return the timezone adjusted date / time.

Reply:
Sure thing